ddtrace/contrib/httpx/__init__.py

```diff
@@ -0,0 +1,67 @@
+"""
+The ``httpx`` integration traces all HTTP requests made with the ``httpx``
+library.
+
+Enabling
+~~~~~~~~
+
+The ``httpx`` integration is enabled automatically when using
+:ref:`ddtrace-run<ddtracerun>` or :ref:`patch_all()<patch_all>`.
+
+Alternatively, use :ref:`patch()<patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(httpx=True)
+
+    # use httpx like usual
+
+
+Global Configuration
+~~~~~~~~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.httpx['service']
+
+   The default service name for ``httpx`` requests. This value will
+   be overridden by an instance override or if the ``split_by_domain`` setting is
+   enabled.
+
+   This option can also be set with the ``DD_HTTPX_SERVICE`` environment
+   variable.
+
+   Default: ``None`` (inherit from ``DD_SERVICE``)
```

Comment:
```suggestion
   Default: ``None``
```

I don't think this is true since `ext_service` is being used

Reply:
Yeah this is really where service naming breaks down 😕. Consistency might be the way to go here just so that the experience is the same between the http clients but I'm ok diverging from the other integrations if we document the expected usage of `httpx` (overriding the service name) for external requests

Reply:
Here are the definitions in the code: https://github.com/DataDog/dd-trace-py/blob/c5b5bd2410da75a0e89f601512999186179a7260/ddtrace/contrib/trace_utils.py#L178-L229

The descriptions on the functions are pretty good, but the tl;dr; is:

`int_service` is a helper to determine the service name for an "internal service" and `ext_service` is to determine the service name for an "external service".

external service - a span which represents a call to a third party (out of process) service
internal service - basically everything else